include/swift/AST/DiagnosticsSema.def

```diff
@@ -3566,6 +3558,23 @@ WARNING(debug_long_closure_body, none,
         "closure took %0ms to type-check (limit: %1ms)",
         (unsigned, unsigned))
 
+//------------------------------------------------------------------------------
+// Pattern match diagnostics
+//------------------------------------------------------------------------------
+
+
+ERROR(empty_switch_stmt,none,
+      "'switch' statement body must have at least one 'case' or 'default' "
+      "block; do you want to add a default case?",())
+ERROR(non_exhaustive_switch,none,
+      "%select{switch must be exhaustive, consider adding |do you want to add }0"
+      "%select{missing cases|a default clause}1"
+      "%select{:|?}0 %2", (bool, bool, StringRef))
```

Comment:
Ah yeah.  That was leftover from when I thought I could combine this fixit with the other one. 

Thanks!  I'll shoot a quick pull request over and cherry-pick that in here.